planning/mission_planner/src/lanelet2_plugins/default_planner.hpp

```diff
@@ -51,11 +59,14 @@ class DefaultPlanner : public mission_planner::PlannerPlugin
   lanelet::ConstLanelets shoulder_lanelets_;
   route_handler::RouteHandler route_handler_;
 
+  DefaultPlannerParameters p;
```

Comment:
Could you rename it to something like `param_` to meet the [guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/)?
- https://google.github.io/styleguide/cppguide.html#Variable_Names
- https://google.github.io/styleguide/cppguide.html#Variable_Comments

Reply:
Thanks for the reviews, I implemented and pushed them all at latest commit.